.mage/dev.go

```diff
@@ -133,6 +215,7 @@ func (Dev) DBRedisCli() error {
 
 // InitStack initializes the Stack.
 func (Dev) InitStack() error {
+	mg.Deps(Dev.DBStart)
```

Comment:
This should not be a dependency - user might want to run the stack with a locally-running database